src/action.d.ts

```diff
@@ -0,0 +1,48 @@
+/**
```

Comment:
can we add the LICENSE header here as well ?

Reply:
Will do.